docs/root/version_history/current.rst

```diff
@@ -32,6 +32,7 @@ New Features
 * hds: added support for delta updates in the :ref:`HealthCheckSpecifier <envoy_v3_api_msg_service.health.v3.HealthCheckSpecifier>`, making only the Endpoints and Health Checkers that changed be reconstructed on receiving a new message, rather than the entire HDS.
 * health_check: added option to use :ref:`no_traffic_healthy_interval <envoy_v3_api_field_config.core.v3.HealthCheck.no_traffic_healthy_interval>` which allows a different no traffic interval when the host is healthy.
 * listener: added an optional :ref:`default filter chain <envoy_v3_api_field_config.listener.v3.Listener.default_filter_chain>`. If this field is supplied, and none of the :ref:`filter_chains <envoy_v3_api_field_config.listener.v3.Listener.filter_chains>` matches, this default filter chain is used to serve the connection.
+* lua: added `downstreamDirectRemoteAddress()` and `downstreamLocalAddress()` APIs :ref:`streamInfo() <config_http_filters_lua_stream_info_wrapper>`.
```

Comment:
Oops, sorry, missing "to"?

lua: added `downstreamDirectRemoteAddress()` and `downstreamLocalAddress()` APIs to :ref:`stream info object <config_http_filters_lua_stream_info_wrapper>`.

Reply:
fixed. thanks!